### ISSUE TITLE
NPS for Navisworks manage 2022

### ISSUE THREAD

**Issue by jr-desy** [dimven/NavisPythonShell#14]
Anyone successfully installed nps on Naviswoks manage 2022?

**Comment by 10-x-10** [comment]
Hi! Is it possible to install NPS to Navisworks Manage 2023?

**Comment by akeelg** [comment]
Hey,
Any updates?

**Comment by chuongmep** [comment]
Hi @akeelg , problem fixed in https://github.com/dimven/NavisPythonShell/pull/15

**Comment by wishbonesr** [comment]
@jr-desy 
Yes, and on 2023.  The readme followed to the 'T', and went ahead and upgraded the framework for the project to 4.8 to be comparable to the version used by the 2023 NW api's.
